### DISCUSSION TITLE
Crazyflie won't flash with AI Deck

### DISCUSSION THREAD

**Discussion by shuosha** [bitcraze/discussions#1596] [General]
We have a Crazyflie 2.0 with an AI deck attached. However, attempting to flash with the Crazyradio gives no results. Neither does flashing it with the usb directly.

**Comment by gemenerik** [comment]
Can you elaborate a little bit? Are you trying to flash the Crazyflie 2.0? What version of the lib and client are you running? Do you get any errors or warnings in the terminal, or does it just freeze, or something else?
